source/aggregation-pipeline-builder.txt

```diff
@@ -89,69 +89,210 @@ Limitations
 Create an Aggregation Pipeline
 ------------------------------
 
-Aggregation pipelines can be created in two seperate modes:
+Aggregation pipelines can be created in three different modes:
 
-- Stage View Mode, a visual click-based pipeline editor that 
-  preloads pipeline syntax based on your selected stages.
+- Stage View Mode, a visual click-based pipeline editor that preloads pipeline 
+  syntax based on your selected stages.
+
+- Focus Mode, a feature of Stage View Mode that allows you to focus on editing
+  one pipeline stage at a time. This can be helpful if you are working with 
+  complex or deeply nested aggregation pipeline stages.
 
 - Text View Mode, a text-based pipeline editor that accepts raw
   pipeline syntax. 
 
-.. _create-agg-pipeline-stageview:
-
-Stage View Mode
-~~~~~~~~~~~~~~~
-|
-
-.. include:: /includes/steps/create-agg-pipeline.rst
-
-.. _create-agg-pipeline-textview:
-
-Text View Mode
-~~~~~~~~~~~~~~
-|
-
-.. procedure::
-   :style: normal
-
-   .. step:: In the aggregation pipeline pane, click the 
-      :guilabel:`</> Text` toggle switch to enable text mode for 
-      pipeline editing.
-
-      .. figure:: /images/compass/agg-builder-textview-toggle-on.png
-         :alt: Toggle textmode on
-
-   .. step:: Enter valid aggregation syntax in the text editor. 
-      The following example pipeline limits the query results to ``4`` 
-      documents.
-
-      .. code-block:: javascript
-
-         [ { "$limit" : 4 } ]
-
-      .. figure:: /images/compass/agg-builder-textview-edit.png
-         :alt: Edit the text aggregation pipeline
-
-   .. step:: **(Optional)** If preview mode is enabled, you can 
-      click :guilabel:`Output Options` in the Pipeline Output pane to 
-      expand all nested fields and documents within the 
-      preview results.
-
-      .. figure:: /images/compass/agg-builder-textview-expand-pipeline-preview.png
-         :alt: Pipeline Output preview
-
-   .. step:: Click :guilabel:`Run` at the top right of the pipeline builder. 
-      |compass-short| returns your results in the document view.
-
-      .. figure:: /images/compass/agg-builder-textview-agg-results.png
-         :alt: Results after Running Aggregation
-
-   .. step:: **(Optional)** Click the :guilabel:`Output Options` dropdown 
-      and select :guilabel:`Expand all fields` to expand all nested 
-      fields and documents within the query results.
-
-      .. figure:: /images/compass/agg-builder-textview-expand-results.png
-         :alt: Expand the aggregation pipeline results
+To see how to create an aggregation pipeline, select the tab corresponding to 
+your view mode of choice:
+
+.. tabs:: 
+
+   .. tab:: Stage View Mode 
+      :tabid: stage 
+
+      .. procedure:: 
```

Comment:
This is not new content — just moved content from the original YAML file into a procedure directive.

Reply:
This content will be updated in a different ticket to reflect the new UI flows

Reply:
Instead, please look at content under the Focus Mode tab